docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
+You can start a Write the Docs Meetup via `Meetup <http://www.meetup.com/>`_. Through Meetup.com, you can create events, contact Meetup members, promote your sponsors, and more. If you need help with Meetup fees, `contact us <mailto:conf@writethedocs.org>`_. We may be able to help.
 
 Create a Meetup Page:
 ~~~~~~~~~~~~~~~~~~~~~
 
-Go to `Meetup website <https://secure.meetup.com/create/>`_ and follow the instructions.
+Go to the `Meetup website <https://secure.meetup.com/create/>`_ and follow the instructions.
 
-a few tips:
+Tips:
 
-* **Topics**: they're keywords associated with your group that will help people find it. If you're not sure what to choose, you can try: Technical Writers, Documentation, Technical Writing, Technical communicators, Technical Documentation Editing, Writing, Document Management Systems, Software Development, Web Development, User Experience, UX Design, Open Source.
-* **Name**: our meetups are usually called "Write the Docs" + city name.
-* **Members' name**: meetup's organizers usually choose "Documentarians".
-* **Code of Conduct**: don't forget to add Write the Docs' :doc:`/code-of-conduct` to the ``page`` section of your meetup.
+* **Topics**: Choose your topics with an eye to the Write The Docs worldview. We want to include everyoen interested in software documentation, so include topics that help publicize your meetups to the developers and engineers in your area, such as Python, JavaScript, APIs, Perl, Ruby, UX, User Experience, as well as standard topics associated with techincal writing.
+* **Name**: Our meetup include the following standard name: "Write the Docs" <city name>, such as "Write The Docs PDX".
+* **Members' name**: Our standard name is "Documentarian".
+* **Code of Conduct**: Do include a code of conduct. For options, see the stadnard Write the Docs' :doc:`/code-of-conduct` or the abbreviated version used by `Write the Docs PDX <https://www.meetup.com/Write-The-Docs-PDX/pages/19618086/Code_of_Conduct/>`_
 
-If you need more help, you can check `Meetup's documentation <http://www.meetup.com/help/topics/10/article/464982/>`_.
+Once you've started your Meetup, `contact us by email <mailto:conf@writethedocs.org>`_ or on `Slack <https://writethedocs.slack.com/>`_: we will add your group to the :doc:`/meetups/index` list on our website.
 
-When you're done, `contact us <mailto:conf@writethedocs.org>`_ or on `Slack <https://writethedocs.slack.com/>`_: we will add your group to the :doc:`/meetups/index` list on our website.
+Find a Place to Meet
+--------------------
 
-Find a Place to Meet (Sponsors)
--------------------------------
+Ideally, you can start by setting up a Meetup at your corporate facility. If your facilities aren't convenient to your potential community of documentarians, find "documentarians" or "community managers" for centrally located companies in your area. You can typically find these people at other technical meetups/conferences in your area.
```

Comment:
How's this?
If your own workplace isn't a good option for a meetup location, ask other local documentarians or community managers for ideas. Where are other meetups held? Conferences? Do you know someone who works for a likely sponsor or whose company might be willing to host your meetup?